compiler/rustc_apfloat/src/lib.rs

```diff
@@ -538,7 +537,6 @@ pub trait Float:
     ///   NaN -> \c IEK_NAN
     ///   0   -> \c IEK_ZERO
     ///   Inf -> \c IEK_INF
-    ///
```

Comment:
Dismissing my ping on the basis of these being trivial comment-only changes (tho I wish we could've just frozen this directory *sigh*).

> If needed, I could try to go through them and only grab the doc comment changes for the core/std/alloc libraries, removing the rest of the changes.

Consider this comment a (tiny) vote for having a PR just for `library/`, the compiler itself in general is more likely to be chaotic, and even have weird snippets in comments etc.

But if people are willing to review the PR as-is, I have nothing against that, to be clear.